### PR TITLE
fixes #74745 earncoin.site

### LINKED ISSUE #74745: earncoin.site
### Issue URL (Ads)
 [https://earncoin.site/M7YThNlt](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fearncoin.site%2FM7YThNlt)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 17
Browser: | Opera
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74745
Other rules for better cleaning for `earncoin.site`.
Added `https://2shrt.com/n6xQG` from my discovery due to fake buttons.

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+earncoin.site##.content
```
Comment (Sergey-Lyapin) [inline EnglishFilter/sections/specific.txt:257]: I think we should not hide this element, btw this rule is dangerous

Screenshot 2021-02-17 at 15 07 06

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/adservers.txt [hunk 2]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||feyl4mh5to.com^
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:8]: Why not `||feyl4mh5to.com^$third-party`?
Unfortunately, I don't see it on my end with US IP. Can you provide a screenshot?

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: @Sergey-Lyapin That's crap, would the person read that while waiting?

File: EnglishFilter/sections/adservers.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: -_- sure

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/adservers.txt:8]: @zloyden
This request blocks by regesp, check here https://2shrt.com/n6xQG

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: > That's crap, would the person read that while waiting?

Not an ad.

File: EnglishFilter/sections/adservers.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: @zloyden It's obsolete now! Let me remove  it.

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt]: Ok, I found it and agree with this rule.

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -276,7 +276,7 @@ yoshare.net##iframe[id^="yoshare.net_"][id$="_DFP"]
 animax.*##[src*="//a.exdynsrv.com/iframe.php"]
 ||animax.*/ax.js
 dekki.com##div[pb-serve-label*="advert"]
-cuts-url.com##.content
+earncoin.site,cuts-url.com##.content
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:279]: It shouldn't be blocked. Not an ad.

File: EnglishFilter/sections/adservers.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt]: Why? @zloyden. It's annoying when one do not check properly and comment. @zloyden Please.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:279]: It's crap though, would you read it while waiting for the link?

File: EnglishFilter/sections/adservers.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/adservers.txt:8]: @wout-codes
Please turn the rule back, f.e. iOS doesn't support regexps so this rule will be useful

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: I did.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Ok removed.

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: It's their description. Yes, it is located aggressively.
So, the only option is to add the rules to the Annoyances filter -> `annoyances.txt` with task line comment.
Something like that:
```
earncoin.site##.box-main > div.faq
earncoin.site##.short div.row > div.text-center > div.content
```

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt]: Ok added.

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt]: @wout-codes 
I've added them

**Pull request merged**